go/vt/vtgate/planbuilder/jointab.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2016, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package planbuilder
+
+import (
+	"strconv"
+
+	"github.com/youtube/vitess/go/vt/sqlparser"
+)
+
+// ListVarName is the bind var name used for plans
+// that require VTGate to compute custom list values,
+// like for IN clauses.
+const ListVarName = "__vals"
+
+// jointab is used for managing join dependencies.
```

Comment:
Can you add a comment on what it actually does? jointab name is kinda obscure...